src/Containers.jsx

```diff
@@ -40,19 +42,84 @@ class Containers extends React.Component {
         }));
     }
 
-    //TODO
-    stopContainer(container) {
-        return undefined;
+    stopContainer(container, timeout) {
+        document.body.classList.add('busy-cursor');
+        if (!timeout) {
+            timeout = 10;
+        }
+        utils.varlinkCall(utils.PODMAN, "io.podman.StopContainer", {name: container.ID, timeout: timeout})
+            .then(reply => {
+                if (!reply.container) {
+                    return;
+                }
```

Comment:
When does that happen? That looks like an error condition that isn't being handled further?